src/Veldrid/MTL/MTLGraphicsDevice.cs

```diff
@@ -8,6 +8,10 @@
 using NativeLibraryLoader;
```

Comment:
I think this may not be needed if the addition below is made everywhere (instead of behind the #if).